### DISCUSSION TITLE
failed to archive /Users/builder/clone/ios/AppName.xcworkspace

### DISCUSSION THREAD

**Discussion by iib0011** [codemagic-ci-cd/codemagic-docs#1795] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

In the build step I have
```bash
xcodebuild -workspace /Users/builder/clone/ios/AppName.xcworkspace -scheme AppName-archivePath build/ios/xcarchive/AppName_gjqj5r5b.xcarchive archive COMPILER_INDEX_STORE_ENABLE=NO DEVELOPMENT_TEAM=4QJN6V8JA4 'CODE_SIGN_IDENTITY=Apple Distribution' CODE_SIGN_STYLE=Manual
```

### Expected results

The build should succeed

### Actual results

It says  error: No profile for team 'TEAM_ID' matching 'PROFILE_NAME' found: Xcode couldn't find any provisioning profiles matching 'TEAM_ID/PROFILE_NAME'. Install the profile (by dragging and dropping it onto Xcode's dock item) or select a different one in the Signing & Capabilities tab of the target editor. (in target 'APP_NAME' from project 'APP_NAME'

### Build id (optional)

6490d191539adce1eedef8c7

**Comment by himesh-cm** [comment]
Hi @iib0011, any specific reason you are using `Xcode 13.3`?
Can you please try the latest version of Xcode?
You can do it in `yaml` as shown below.
`xcode: 14.3`

**Reply by iib0011** [reply]
I was using 14.3. I had the same error but without the cause. Once I swiched to 13.3 I got the cause which I put in the description of the issue

**Reply by himesh-cm** [reply]
Can you please share the build id for Xcode 14.3?

**Reply by iib0011** [reply]
@himesh-cm 64918ec9b7be1028fa9fefce

**Reply by himesh-cm** [reply]
Hi @iib0011 , Can you share the path to your `info.plist` file from your project directory?
Also, the error with 14.3 is different.

**Reply by iib0011** [reply]
@himesh-cm the path is ios/app_name/Info.plist

**Reply by himesh-cm** [reply, marked as answer]
@iib0011, `Set info.plist` step is failing, please correct the path as you mentioned and try re-running the build.

**Reply by iib0011** [reply]
What should the path be?

**Reply by iib0011** [reply]
It is currently ios/app_name/Info.plist

**Reply by iib0011** [reply]
@himesh-cm Thank you. You are right

**Reply by iib0011** [reply]
Can you check why it is still failing? This is the build id: 6495f46fdd02b8948a39e43d

**Reply by himesh-cm** [reply]
Hi @iib0011, Are you still looking for assistance?

**Reply by iib0011** [reply]
I'm debugging. I'll let you know if I need help

**Comment by himesh-cm** [comment]
While I’m closing our thread for now, if you have any other questions or suggestions, feel free to reply back. Have a nice day!

**Reply by iib0011** [reply]
Hi @himesh-cm . Can you plz check why this build is failing? 64a33b35f961fa574934c1c6
